02_bubble/pokkalar/main_test.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"bytes"
+	"reflect"
+	"testing"
+)
+
+var testData = map[string]struct{ input, expected []int }{
+	"Empty Array":        {[]int{}, []int{}},
+	"Single Digit Array": {[]int{1}, []int{1}},
+	"Standard Input":     {[]int{5, 8, 7}, []int{5, 7, 8}},
+	"Negative Input":     {[]int{-4, -6, -2}, []int{-6, -4, -2}},
+	"Long Set":           {[]int{12, 15, 8, 5, 3, 2, 4, 1, 0}, []int{0, 1, 2, 3, 4, 5, 8, 12, 15}},
```

Comment:
While we are here, you could add a test array for repeated elements

Reply:
Added Repetitive set